thoth/adviser/utils.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# thoth-adviser
+# Copyright(C) 2020 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Various helpers and utility functions."""
+
+import os
+
+from typing import Any
+
+
+def should_keep_history(value: Any) -> bool:
+    """Check if the history should be kept.
+
+    Used as click's converter. If not set explicitly during invocation, check
+    environment variable to turn of history tracking.
+    """
+    if value is None:
```

Comment:
maybe its good to mention the default value and its dependency on the ENV in the docstring?!

Reply:
Done in 859f56c